src/components/Card/ProductCard/ProductCard.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+
+function ProductCard({badge,img,description,isBtnSaveToWishlist=false,originalPrice,name,discountedPrice,discountPercent,icon}) {
```

Comment:
use prettier for code format, your lines are too lengthy
and also not formatted

Reply:
I am actually using beautify.